src/Api/FingerprintApi.php

```diff
@@ -85,6 +85,9 @@ public function getConfig()
     public function getEvent($request_id)
     {
         list($response) = $this->getEventWithHttpInfo($request_id);
+        if ($response === null) {
+            throw new \Exception("SerializationError, please get full response body with getEventWithHttpInfo");
```

Comment:
Let's introduce custom Exception class with this error, it will be easier for customers to handle it.